<!DOCTYPE HTML>
<!--
	Photon by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Absolute quantification of mtDNA</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->

		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="stylesheet" href="assets/css/custom.css" />
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
	</head>
	<body>
			<nav class="navbar navbar-default navbar-fixed-top navbar-inverse">
			  <div class="container-fluid">
			  <ul class="nav navbar-nav navbar-left">
			  	<li><a href="index.html#landing_page">
			  		<i class="fa fa-line-chart fa-2x" aria-hidden="true"></i>
			  		</a>
			  	<li>
			  </ul>
	          <ul class="nav navbar-nav navbar-right">
		        <li><a href="index.html"><span>INTRO<span></a></li>
		        <li><a href="background.html"><span>BACKGROUND<span></a></li>
		        <li><a href="model.html"><span>MODEL<span></a></li>
		        <li><a href="inference.html"><span>INFERENCE<span></a></li>
		        <li><a href="results.html"><span>RESULTS<span></a></li>
		        <li><a href="summary.html"><span>SUMMARY<span></a></li>
		      </ul>
			  </div>
			</nav>

		<!-- One -->
			<section id="one" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Quantitative Polymerase Chain Reaction (qPCR)</h2>
								<p>Specific details of qPCR may be abstracted away from for the purposes of this project. However, the basic aspects and important terms such as fluorescence, primers, efficiency and amplification phases are necessary in understanding the mathematical model and are explained below.
							</header>
						</div>
						<div class="6u 12u$(medium)">
							<h3>Basics of qPCR</h3>
							<p>Quantitative polymerase chain reaction (qPCR) is a technique used to amplify a set of DNA segment copies. It is useful for measuring the initial molecule copy number when it is too small to be determined experimentally without amplification.</p>
							<p>
							The molecules undergo a series of thermal cycles. During every cycle, the following steps occur:
							<ol>
								<li>Heating causes the double DNA strand to be split into two separate strands.</li>
								<li>Primers designed to target a specific DNA fragment are introduced and bind to the desired segments.</li>
							</ol>
						</div>
						<div class="6u$ 12u$(medium)">
							<img src="diagrams/pcr_amplification.png" style="width:100%" alt="mtDNA diagram">
							<p><i>Placeholder; need copyrights for the image</i></p>
						</div>
						<div class="12u$ 12u$(medium)">
							</p>
							A primer is a short segment of single-stranded complimentary DNA designed to bind to a specific DNA sequence. Different primers are used to amplify mutant mtDNA molecules and any mtDNA molecule.
							<p><br>*discuss probes; efficiency*</p>
							<h3>The amplification curve</h3>
							<p>Fluorescence intensities recorded after each qPCR cycle form a sigmoidal amplification curve. Initially, the molecules amplify at peak efficiency (close to doubling at each cycle), showing an exponential increase in fluorescence. After ~35 cycles, the reaction starts to slow down and the efficiency decreases. Finally, the reaction stops and no molecules are amplified, causing the fluorescence intensity to be close to constant between consecutive cycles. Therefore, the amplification curve can be divided into three phases: exponential, linear and plateau. 
							</p>
							

							<div id="qpcr_plot" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<p><i>The above figure shows the three phases of the amplification curve: exponential, linear and plateau. Information from the exponential phase is used for the inference. </i></p>

							<p>The mathematical model used to represent the qPCR amplification assumes that the efficiency is constant. Therefore, before running the inference method, the exponential phase information is extracted from the amplification curve, and the rest of the fluorescence reads discarded.</p>
							*Empirical AIC for checking whether sigmoid or exp fits better*
						</div>
					</div>
				</div>
			</section>

		<!-- Two -->
		<section id="one" class="main style2">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Experimental protocol</h2>
								<p>The experimental protocol for measuring heteroplasmy in a single cell consists of quantifying the mutant and total mtDNA load separately using qPCR. The qPCR amplification curves of the two experiments are modelled separately and used to infer the initial quantities. The heteroplasmy is the ratio between mutant and total mtDNA quantities.</p>
							</header>
						</div>
						<div class="6u 12u$(medium)">
							<p>A qPCR experiment can only quantify one type of molecule, as it uses a primer to identify a specific sequence of the target molecule. The two molecule types of interest are the mutant and the general mtDNA molecule, with the latter including both mutant and wildtype molecules.</p>
							<p>The experimental protocol aims to equally divide the contents of a single cell into two separate solutions. One solution is used to infer the amount of mutant (m), while the other is used to infer the amount of total mtDNA (N). The assumption is that after the cell is split, both solutions contain half of the mutant and half of the total number of mtDNA molecules.</p>
							<p>For each of the two solutions, a qPCR primer is used to identify and amplify either a DNA segment specific to mutant mtDNA, or a segment present in any mtDNA molecule.</p>
							<p> The resulting data consists of amplification curves for each of the two solutions. We use this data to estimate and model the measurement error in the initial number of molecules m/2 and N/2.</p>
						</div>
						<br/>
						<div class="6u$ 12u$(medium) important(medium)">
							*amplification chart*
						</div>
					</div>
				</div>
			</section>
			<section id="three" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="6u 12u$(medium)">
							Example plot from standard curve method data.
						</div>
						<div class="6u$ 12u$(medium)">
							<header class="major">
								<h2>The standard curve method</h2>
							</header>
							<p>About standard curve method with Ct and limitations.</p>
						</div>
					</div>
				</div>
			</section>

		<!-- Three -->
			<section id="four" class="main style2">
				<div class="container">
					<div class="row 150%">
						<div class="6u 12u$(medium)">
							<header class="major">
								<h2>Single-experiment inference</h2>
							</header>
							<p>Basics</p>
							<p>What data we use (exponential phase)</p>
							<p>Model described in more detail on the next page</p>
						</div>
						<div class="6u$ 12u$(medium) important(medium)">
						</div>
					</div>
				</div>
			</section>

		<!-- Footer -->
			<section id="footer">
				<ul class="copyright">
					<li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
				</ul>
			</section>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>
			<!-- High charts -->
			<script src="https://code.highcharts.com/highcharts.js"></script>
			<script src="https://code.highcharts.com/modules/exporting.js"></script>
			<script src="background.js"></script>

	</body>
</html>