<!DOCTYPE HTML>
<!--
	Photon by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Absolute quantification of mtDNA</title>
		<link rel="shortcut icon" href="diagrams/title-icon.ico">
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" integrity="sha384-BVYiiSIFeK1dGmJRAkycuHAHRg32OmUcww7on3RYdg4Va+PmSTsz/K68vbdEjh4u" crossorigin="anonymous">
		<!--[if lte IE 8]><script src="assets/js/ie/html5shiv.js"></script><![endif]-->

		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="stylesheet" href="assets/css/custom.css" />
		<!--[if lte IE 8]><link rel="stylesheet" href="assets/css/ie8.css" /><![endif]-->
		<!--[if lte IE 9]><link rel="stylesheet" href="assets/css/ie9.css" /><![endif]-->
	</head>
	<body>
			<nav class="navbar navbar-default navbar-fixed-top navbar-inverse">
			  <div class="container-fluid">
			  <ul class="nav navbar-nav navbar-left">
			  	<li><a href="index.html#landing_page">
			  		<i class="fa fa-line-chart fa-2x" aria-hidden="true"></i>
			  		</a>
			  	<li>
			  </ul>
	          <ul class="nav navbar-nav navbar-right">
		        <li><a href="index.html"><span>INTRO<span></a></li>
		        <li><a href="background.html"><span>BACKGROUND<span></a></li>
		        <li><a href="model.html"><span>MODEL<span></a></li>
		        <li><a href="inference.html"><span>INFERENCE<span></a></li>
		        <li><a href="results.html"><span>RESULTS<span></a></li>
		        <li><a href="summary.html"><span>SUMMARY<span></a></li>
		        <li><a href="references.html"><span>REFERENCES<span></a></li>
				<li><a href="glossary.html"><span>GLOSSARY</span></a></li>
		      </ul>
			  </div>
			</nav>

		<!-- One -->
			<section id="qpcr" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Quantitative Polymerase Chain Reaction (qPCR)</h2>
								<p>Specific details of qPCR may be abstracted away from for the purposes of this project. However, the basic aspects and important terms such as fluorescence, primers, efficiency and amplification phases are necessary in understanding the mathematical model and are explained below.
							</header>
						</div>
						<div class="6u 12u$(medium)">
							<h3>Basics of qPCR</h3>
							<p>Quantitative polymerase chain reaction (qPCR, <a href="references.html#heid">Heid et al., 1996</a>) is a technique used to amplify a set of DNA segment copies. It is useful for measuring the initial molecule copy number when it is too small to be determined experimentally without amplification.</p>
							<p>
							The molecules undergo a series of thermal cycles (Fig. 6). During every cycle, the following steps occur:
							<ol>
								<li>Heating causes the double DNA strand to be split into two separate strands.</li>
								<li>Primers designed to target a specific DNA fragment are introduced and bind to the desired segments. Special fluorescent reporter probes also bind to the desired segment, but at the other end than the primer.</li>
								<li>Polymerisation starts from the primer and goes along the DNA segment to "reconstruct" the complementary strand.</li>
								<li>When polymerisation ends, it reaches a reporter probe, degrades it and causes a fluorescent reporter to be emitted, which in turns increases fluorescence.</li>
							</ol>
						</div>
						<div class="6u$ 12u$(medium)">
							<center><img src="diagrams/qpcr_amplification.svg" style="width:80%" alt="mtDNA diagram"></center>
							<figcaption><center><b>Figure 6.</b> The qPCR experiment amplifies a set of DNA fragments using primers designed to identify and bind to these fragments. The molecules are amplified over a set of thermal cycles and a fluorescence intensity is reported after each cycle.</center></figcaption>
						</div>
						<div class="12u$ 12u$(medium)">
							<p>
							A primer is a short segment of single-stranded complimentary DNA designed to bind to a specific DNA sequence. Different primers are used to amplify mutant mtDNA molecules and any mtDNA molecule. Similarly to a primer, a probe is a short segment of single-stranded DNA design to bind to a specific DNA sequence. When polymerisation ends, it releases a reporter that causes an increase in fluorescence.
							</p>
							<p>
							The efficiency of a qPCR experiment dictates how many molecules are amplified during each cycle. Is dependent on both the primers and the DNA template, and is an important aspect when interpreting qPCR amplification data, as we discuss below.
							</p>
							<h3>The amplification curve</h3>

							<p>Fluorescence intensities recorded after each qPCR cycle form a sigmoidal amplification curve. The inference method uses a mathematical model with a set of parameters to represent the amplification curve. It infers the parameters that best fit the model to the amplification data.
							</p>

							<p>At the start of the experiment, the molecules amplify at peak efficiency (close to doubling at each cycle), showing an exponential increase in fluorescence (<i>exponential phase</i>). Then, the reaction starts to slow down and the efficiency decreases (<i>linear phase</i>). Finally, the reaction stops and no molecules are amplified, causing the fluorescence intensity to be close to constant between consecutive cycles (<i>plateau phase</i>). Therefore, the amplification curve can be divided into three phases: <i>exponential</i>, <i>linear</i> and <i>plateau</i> (Fig. 7). 
							</p>
							

							<div id="qpcr_plot" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<figcaption><center><b>Figure 7.</b>The amplification curve consists of three phases: exponential (peak, constant efficiency), linear (decrease in efficiency) and plateau (efficiency 0, reaction stops). Only the information from the exponential phase, when efficiency is constant, is used for the inference. </center></figcaption>
						</div>
					</div>
				</div>
			</section>

		<!-- Two -->
		<section id="protocol" class="main style2" style="margin-top:-70px; padding-top:70px">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Experimental protocol</h2>
								<p>The experimental protocol for measuring heteroplasmy in a single cell consists of quantifying the mutant and total mtDNA load separately using qPCR. The qPCR amplification curves of the two experiments are analysed separately and used to infer the initial quantities. Heteroplasmy is the ratio between mutant and total mtDNA quantities.</p>
							</header>
						</div>
						<div class="12$ 12u$(medium)">
							<p>A qPCR experiment can only quantify one type of molecule, as it uses a primer to identify a specific sequence of the target molecule. The two molecule types of interest are the mutant and the general mtDNA molecule, with the latter including both mutant and wildtype molecules.</p>
						</div>

						<div class="6u 12u$(medium)">
							<p>The experimental protocol aims to equally divide the contents of a single cell into two separate samples, each containing a half of the mutant load (m/2) and a half of the total load (N/2). One sample is used to infer the initial amount of mutant (m), while the other is used to infer the initial amount of total mtDNA (N). The assumption is that the initial solution is well mixed and after it is split, both samples contain half of the mutant and half of the total number of mtDNA molecules.</p>
							<p>For each of the two solutions, a qPCR primer is used to identify and amplify either a DNA segment specific to mutant mtDNA, or a segment present in any mtDNA molecule.</p>
							<p>A qPCR machine consists of a plate with 384 wells, allowing a maximum of 384 samples to be analysed in parallel. When the contents of a single cell are split into two halves, the resulting samples are placed on two wells of the same qPCR plate. The fluorescence intensity reads from the qPCR experiment consist of two amplification curves. We use this data to estimate and model the measurement error in the initial number of molecules m/2 and N/2 (Fig. 8).</p>
						</div>
						<br/>
						<div class="6u$ 12u$(medium) important(medium)">
							<center><img src="diagrams/qpcr_plate.svg" style="width:70%" alt="Protocol diagram"></center>
							<figcaption><center><b>Figure 8.</b> Heteroplasmy is quantified by mixing cell contents, splitting in two halves and placing them onto separate qPCR wells. A qPCR experiment is used to amplify a sequence specific to the mutation in one well, while the other is used to amplify the a sequence specific to any mtDNA molecule.</center></figcaption>
						</div>
					</div>
				</div>
			</section>
			<section id="three" class="main style1">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>The standard curve method</h2>
								<p>The standard curve method is the current go-to method in absolute quantification of target analytes from qPCR data. It is easy to use, but it oversimplifies the qPCR process, doesn't account for stochasticity and fails to provide an error model for the estimate. We first discuss the standard curve method before motivating the efforts of using Bayesian inference instead.
							</header>
						</div>
						<div class="6u 12u$(medium)">
							<p>The standard curve method is a technique for building a calibration curve from samples with known properties. In the case of qPCR, the standard curve method is called comparative C<sub>T</sub> (<a href="references.html#schmittgenlivak">Schmittgen and Livak, 2008</a>). It uses a set of samples (standards) where the initial number of molecules X<sub>0</sub> is known. For each standard, the cycle i where the fluorescence F<sub>i</sub> exceeds a fixed threshold T is recorded. A calibration curve is constructed from the (X<sub>0</sub><sup>j</sup>, C<sub>T</sub><sup>j</sup>) for each standard j.
							</p>
							
							<p>Figure 9 shows an example calibration curve constructed from real experimental data. The standard is a dilution series where the initial copy number X<sub>0</sub> is progressively diluted in factors of 10, from 1,000,000 to 100. The calibration curve is linear when plotting the logarithm of the copy number, under the assumption that efficiency is constant.
							</p>
							<p>The calibration curve is used to estimate the unknown initial number of molecules X<sub>0</sub> for new samples by using the observed C<sub>T</sub> value (Fig. 9).</p>
						</div>
						<div class="6u$ 12u$(medium)">
							<div id="standard_curve" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<figcaption><center><b>Figure 9.</b> The C<sub>T</sub> calibration curve built from standards with known initial copy number is used to estimate the inital copy numbers for new samples using the C<sub>T</sub> value, the fractional cycle at which fluorescence exceeds a threshold T.</center></figcaption>
						</div>
						<div class="6u 12u$(medium)">
							<h3>Disadvantages</h3>
							<p>One of the main disadvantages of the standard curve method is that it assumes the amplification efficiency between the standards used to construct the curve and the analysed samples is the same.</p>
							<p>Another disadvantage is that the standard curve method does not model the qPCR amplification as a stochastic process. For small numbers of initial molecules (X<sub>0</sub>~100) and efficiency smaller than 1, which is the case for single-cell mtDNA, the amplification process shows stochasticity which is not captured by the standard curve method (Fig. 10). For a stochastic process, a sample with the same X<sub>0</sub> may yield different C<sub>T</sub> values under different runs. Therefore, both a stochastic model and more information than just a point on the amplification curve are needed for an accurate estimate of the copy number X<sub>0</sub>.</p>
						</div>
						<div class="6u$ 12u$(medium)">
							<div id="stochasticity2" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<figcaption><center><b>Figure 10.</b> For small values of initial copy numbers, qPCR amplification is a stochastic process. The high variability in amplified molecules at each cycle is demonstrated by simulating 1,000 experiments in-silico with X<sub>0</sub> = 50, efficiency r = 0.75 and showing the 5-95% confidence interval at each cycle.</center></figcaption>
						</div>
					</div>
				</div>
			</section>

		<!-- Three -->
			<section id="four" class="main style2">
				<div class="container">
					<div class="row 150%">
						<div class="12u$ 12u$(medium)">
							<header class="major">
								<h2>Single-experiment inference</h2>
								<p>The standard curve method discussed above uses a single point on the amplification curve to estimate the inital target quantity X<sub>0</sub>. It does not provide an error model and fails to account for the inherent stochasticity of the qPCR process. This projects extends the work of <a href="references.html#lalam">Lalam, 2007</a> and uses Bayesian inference on the data from the entire amplification curve to determine X<sub>0</sub>. Most importantly, it provides error bars that capture our belief in the copy number estimate, which is essential for interpreting experimental results.
								</p>
							</header>
							<p>Single-experiment inference uses as data the fluorescence intensity reads from the amplification curve of a sample with unknown initial copy number X<sub>0</sub>. By looking at the entire amplification curve rather than just the C<sub>T</sub> value, it can infer the efficiency of a sample without assuming it is the same as that of a standard. 
							</p>
						</div>
						<div class="6u 12u$(medium)">
							<h3>The model and inference method</h3>
							<p>A stochastic model to capture the amplification process. This mathematical model is a Hidden Markov Model parameterised by the efficiency r, the initial number of molecules X<sub>0</sub> and the noise variance in fluorescence reads σ<sup>2</sup>. More details and a motivation for the choice of model can be found in <a href="model.html">the Model section</a>.</p>
							<p>The parameters of model are infered using the pseudo-marginal Metropolis Hastings algorithm. The algorithm uses observation data (fluorescence intensity reads) to sample from the joint posterior distribution of the model parameters. Most importantly, one of the parameters of the model is the initial copy number X<sub>0</sub>, the quantity of interest. Knowing the shape of the posterior distribution for X<sub>0</sub> gives both an estimate of the initial copy number and an error bar on this estimation (Fig. 11). More details about what a posterior distribution is and inference in general can be found in <a href="inference.html">the Inference section</a>.</p>
						</div>

						<div class="6u$ 12u$(medium)">
							<div id="x0_posterior" style="min-width: 310px; height: 400px; margin: 0 auto"></div>
							<figcaption><center><b>Figure 11.</b> The inference algorithm produces samples from the posterior distribution of parameters of the model given the fluorescence data. Most importantly, it produces samples of the initial copy number X<sub>0</sub>. These samples put an error bar on the copy number estimate and capture our belief in experimental results.
							</center>
							</figcaption>
						</div>
					</div>
				</div>
			</section>

		<!-- Footer -->
			<section id="footer">
				<ul class="actions">
					<li><a href="model.html" class="button scrolly">Read next section</a></li>
				</ul>
				<ul class="copyright">
					<li>© 2017 Daria Dicu</li>
					<li>Imperial College London</li>
					<li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
				</ul>
			</section>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>
			<!-- High charts -->
			<script src="https://code.highcharts.com/highcharts.js"></script>
			 <script src="https://code.highcharts.com/highcharts-more.js"></script>
			<script src="https://code.highcharts.com/modules/exporting.js"></script>
			<script src="background.js"></script>

	</body>
</html>